src/main/scala/tech/sourced/gemini/FileQuery.scala

```diff
@@ -13,6 +13,7 @@ import tech.sourced.featurext.generated.service.FeatureExtractorGrpc.FeatureExtr
 import tech.sourced.gemini.util.MathUtil
 
 import scala.collection.JavaConverters._
+import scala.collection.JavaConversions.mapAsScalaMap
```

Comment:
It's better to avoid using this old conversions API and stick to a new one from `JavaConverters` imported above instead \w it's `.asScala` .